Tasks/FtpUpload/ftpuploadtask.ts

```diff
@@ -128,8 +128,16 @@ function doWork() {
         }
     })
 
+    let verboseSnippet: string[] = [];
+    let debugLogger:any = function(message) {
+        verboseSnippet.push(message);
+        if (verboseSnippet.length >= 5) {
```

Comment:
So you only want the last 5 messages? That seems like a small number.

Reply:
It will always print on 'debug'.  This is for non-debug && error.